### PR TITLE
Add current values from libsensors.

### PR DESCRIPTION
**Pull request by sKuhne**
Four sensors are available with this patch.
Tested with axp209-isa-0000 on BananaPi-R1.

### CODE REVIEW

File: src/sensors.c [hunk 1]
```diff
@@ -131,6 +133,10 @@ static sensors_labeltypes_t known_features[] =
 	{ "2.0V", SENSOR_TYPE_VOLTAGE },
 	{ "12V", SENSOR_TYPE_VOLTAGE },
 	{ "power1", SENSOR_TYPE_POWER }
+	{ "curr1", SENSOR_TYPE_CURRENT }
```
Comment (mfournier) [inline src/sensors.c]: Older gcc versions seem to choke on this: `sensors.c:136: error: expected '}' before '{' token`. See the report from the jenkins build.

**Comment by mfournier** [comment]
Thanks for the update @sKuhne !

I was taking a look at the pending patches to sensors.c, and noticed some overlap with #1491. What do you think ? Would it make sense to merge @dangowrt's patch into yours ?

Also, @rubenk mentions these symbols were added in API version 0x402. Could you please make the patchset safe for older versions ? Thanks !

**Comment by sKuhne** [comment]
@mfournier:
The other patch add one value more.
I can add this to my patch and add the test for older versions. Is this okay for you?

**Comment by mfournier** [comment]
That would be awesome @sKuhne !

As I don't have very interesting hardware (from libsensors' perspective) to test this, it would be great if you can check @dangowrt's patch works as expected.

**Comment by sKuhne** [comment]
I've add #1491 and add the 0x402 check. But I can't test energy.

File: src/sensors.c [hunk 2]
```diff
@@ -64,9 +64,7 @@ static char *sensor_type_name_map[] =
 	"temperature",
 # define SENSOR_TYPE_POWER       3
 	"power",
-# define SENSOR_TYPE_CURRENT     4
-	"current",
-# define SENSOR_TYPE_UNKNOWN     5
```
Comment (rubenk) [inline src/sensors.c]: Removing this breaks older api's that don't have SENSOR_TYPE_CURRENT.

Comment (sKuhne) [inline src/sensors.c]: I didn't check the complete "#if" part.
This code is only for API below 0x400 and this does not have this feature as I see in other comments. So I have to clean "sensors_labeltypes_t".

Comment (rubenk) [inline src/sensors.c]: But you are using SENSOR_TYPE_CURRENT unguarded in known_features[]. If it is not defined, the code breaks. Please check your code against an older version of libsensors.

Comment (rubenk) [inline src/sensors.c]: You can see this issue here: https://ci.collectd.org/job/pull-requests-build-on-epel5-x86_64-with-default-toolchain/812/console, it indeed breaks on RHEL5.

**Comment by clemensg** [comment]
I created #2255

**Comment by rubenk** [comment]
Should be fixed by b2b559936dd8e1c476257021add9310c414e4df4

**Pull request closed without merging**